samples/csharp/MicrosoftGraphSDKBeta/Team.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Graph;
+using Microsoft.Graph.Beta;
+using Microsoft.Graph.Beta.Models;
+
+namespace MicrosoftGraphSDK
+{
+    /// <summary>
+    /// Graph SDK endpoints for teams
+    /// </summary>
+    public static class Team
+    {
+        /// <summary>
+        /// Returns teams information
+        /// </summary>
+        /// <param name="client">Microsoft Graph service client</param>
+        /// <returns>TeamCollectionResponse</returns>
+        public static async Task<TeamCollectionResponse> GetJoinedTeamsAsync(
+            this GraphServiceClient client)
+        {
+            try
+            {
+                return await client.Me.JoinedTeams
+                 .GetAsync();
+            }
+            catch (Exception ex)
+            {
+                throw new GraphException($"GetJoinedTeamsAsync call: {ex.Message}");
```

Comment:
consider: do we need to be `catch`ing and re`throw`ing in all these methods? seems like it'd be reasonable to just allow the `Exception` from `.GetAsync()` to bubble up...or are you trying to illustrate that `.GetAsync()` can throw? If so, might be useful to express all the Exception types it can throw.

Reply:
@KevinH-MS  basically, we just wanted to show a way to include all possible info (params, method name & exception) from the exception, and this practice can help the customer to identify and troubleshoot efficiently any possible problem coming from SDK in their apps.